src/game_updater/GameUpdater.cpp

```diff
@@ -67,33 +185,11 @@ void GameUpdater::Update(std::shared_ptr<GameState> game_state) {
   if (dX < MIN_DELTA_X) {
     dX = MIN_DELTA_X;
   }
-  float player_y_pos = max(player->GetPosition()[1] + dY
-                              + player->GetVerticalVelocity(),
-                              current_platform_position.y + dY + 0.75f);
-  camera->setPosition(glm::vec3(camera->getPosition()[0] + dX,
-                                camera->getPosition()[1] + dY,
-                                camera->getPosition()[2]));
-  camera->setLookAt(glm::vec3(camera->getLookAt()[0] + dX,
-                              camera->getLookAt()[1] + dY,
-                              camera->getLookAt()[2]));
-
-  player->SetPosition(glm::vec3(player->GetPosition()[0] + dX,
-                                player_y_pos,
-                                player->GetPosition()[2]));
-  if (player_y_pos > current_platform_position.y + dY + 0.75f) {
-    player->SetVerticalVelocity(player->GetVerticalVelocity() - Player::GRAVITY);
-  } else {
-    player->SetVerticalVelocity(0);
-  }
 
-  if (level_updater->Done()) {
-    done = true;
-    game_state->SetDone(done);
-  }
-}
+  camera->setPosition(camera->getPosition() + glm::vec3(dX, dY, 0));
```

Comment:
looks like this isnt working just because the player is now moving like 3 or 4 times faster than it was before, but the camera is [trying] to move the same speed as before, so it recenters on the player every time it gets past the current platform